test/contrib/mqtt.uts

```diff
@@ -55,6 +55,9 @@ assert(connect.klive == 60)
 assert(connect.clientIdlen == 17)
 assert(connect.clientId == b'mosqpub/1440-kali')
 
+= MQTTDisconnect
+dc = MQTT()/MQTTDisconnect()
```

Comment:
I think that this should be rewritten as :
```
raw = raw(MQTT()/MQTTDisconnect())
dc = MQTT(raw)
assert MQTTDisconnect in dc
```

Reply:
I wrote the test you suggested but errors still remain

Reply:
I wrote it without testing, it shoud be:
```
r = raw(MQTT()/MQTTDisconnect())                                                                                           
dc= MQTT(r)                                                                                                                
assert dc.type == 14 
```

You also need to fix the issues reported by @gpotter2

Reply:
I fixed issues, still an error on the coverage but no idea on how to fix it

Reply:
Coverage will be fixed if you rebase against master

Reply:
@gpotter2 i run the command `git rebase master` but it returns to me `Current branch master is up to date.`.
How can I fix this?